platform/ios/src/MGLMapView.mm

```diff
@@ -192,15 +187,12 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingState) {
 #pragma mark - Private -
 
 @interface MGLMapView () <UIGestureRecognizerDelegate,
-                          GLKViewDelegate,
                           MGLLocationManagerDelegate,
                           MGLSMCalloutViewDelegate,
                           MGLCalloutViewDelegate,
                           MGLMultiPointDelegate,
                           MGLAnnotationImageDelegate>
 
-@property (nonatomic, readwrite) EAGLContext *context;
```

Comment:
On iOS, `MGLOpenGLStyleLayer.context` depends on this property, which is still declared in MGLMapView_Private.h.